### PR TITLE
fix(sdk): add pool utilization

### PR DESCRIPTION
**Pull request by amateima**
**Motivation**

https://app.shortcut.com/uma-project/story/3190/add-pool-utilization-to-the-sdk

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/sdk/src/across/clients/bridgePool.ts [hunk 1]
```diff
@@ -83,13 +84,17 @@ class PoolState {
...
     // typechain does not have complete types for call options, so we have to cast blockTag to any
-    const exchangeRatePrevious = await this.contract.callStatic.exchangeRateCurrent({
-      blockTag: previousBlock || latestBlock - 1,
-    } as any);
+    const [exchangeRatePrevious, liquidityUtilizationCurrent] = await Promise.all([
+      this.contract.callStatic.exchangeRateCurrent({
+        blockTag: previousBlock || latestBlock - 1,
+      } as any),
+      this.contract.callStatic.liquidityUtilizationCurrent(),
+    ]);
```
Comment (amateima) [inline packages/sdk/src/across/clients/bridgePool.ts:92]: My question is which clients perform these calls because they require gas to be paid

Comment (mrice32) [reply packages/sdk/src/across/clients/bridgePool.ts:92]: What do you mean?

It's not read-only, but that's just a quirk of how the contract works. I think your callStatic accounts for this, right?

**Review by mrice32 — APPROVED** [review]
LGTM!

File: packages/sdk/src/across/clients/bridgePool.ts (hunk 1 above)
Comment (amateima) [reply packages/sdk/src/across/clients/bridgePool.ts:92]: Yep, you're right

**Review by daywiss — APPROVED** [review]

**Pull request merged**